### PR TITLE
Update product-technical-writing-team-handbook.md

### PR DESCRIPTION
**Pull request by justinegeffen**
Added missing links, revised community contribution sections for clarity, minor punctuation edits.

### CODE REVIEW

File: operations/research-and-development/product/product-technical-writing-team-handbook.md [hunk 1]
```diff
@@ -104,13 +104,13 @@ For less urgent work, if no reply has been received within 48 hours, please ping
...
 
-If your documentation request/issue applies to a repository other than mattermost/docs, you can use the Editor Review label and/or ping @justinegeffen.
+If your documentation request/issue applies to a repository other than mattermost/docs, you can use the \*\*Editor Review\*\*label and/or ping @justinegeffen or @amyblaise.
 
 ## General Contribution Workflow
 
 ### Jira Tickets
 
-The Technical Writing Jira board is located here: [https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51). Feature and release writing is planned three sprints in advance. This ensures that ad-hoc work can be taken on. 
+The Technical Writing Jira board is located here:c[https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51](https://mattermost.atlassian.net/secure/RapidBoard.jspa?rapidView=51). Feature and release writing is planned three sprints in advance. This ensures that ad-hoc work can be taken on.
```
Comment (jasonblais) [inline operations/research-and-development/product/product-technical-writing-team-handbook.md]: Did you intend to add the `c` here?

Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: Erroneous typo, thanks for catching it. I've fixed it!

**Review by jasonblais — APPROVED** [review]
Thank you!

File: operations/research-and-development/product/product-technical-writing-team-handbook.md (hunk 1 above)
Comment (justinegeffen) [reply operations/research-and-development/product/product-technical-writing-team-handbook.md]: @amy

**Pull request merged**